src/MakersOfDenmark.Domain/Models/User/MODRole.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MakersOfDenmark.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MakersOfDenmark.Domain.Models.User
+{
+    public class MODRole : Entity<int>
+    {
+        public MODUser User { get; set; }
+        public MakerSpace MakerSpace { get; set; }
+        public MakerSpaceRoles  Role { get; set; }
+        public DateTimeOffset CreatedAt { get; set; } = DateTimeOffset.UtcNow;
```

Comment:
Jeg vil faktisk mene at vi skal abstrahere CreatedAt ud i TypeConfigurationen som en shadow property.

Det skal være en værdi vi aldrig kan tilgå, andet end via. det meget specifikke API EF Core tilbyder til Shadow Properties.

Egentligt synes jeg faktisk det burde være på alle modeller vi gemmer i databasen. 

Hvis vi skal køre med det i koden skal CreatedAt ihvertfald være read-only og ligge på Entity<T> klassen.

Reply:
For enkelthedens (?) skyld, skal den måske over i Entity-klassen

Reply:
Enig